.travis.yml

```diff
@@ -5,13 +5,12 @@ branches:
   only:
     - master
 before_install:
-  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+  - wget https://repo.continuum.io/miniconda/Miniconda3-4.3.21-Linux-x86_64.sh
```

Comment:
I see the build is failing when we use the latest Miniconda.  I think we should be safe to merge and then confirm the merged version runs on Travis CI.